.dockerignore

```diff
@@ -1 +0,0 @@
-src/public
```

Comment:
Should at least contain `./node_modules` folder?

Reply:
I decided to do not use .dockerignore for now

Reply:
Why not? Without a correct .dockerignore, things like `node_modules`, .git etc. just bloat the build context and lead to unnecessary cache invalidations.

Having a correct .dockerignore is [considered a best-pracitce](https://docs.docker.com/build/building/best-practices/#exclude-with-dockerignore) and I think we should stick to that.

Reply:
done